sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -101,6 +103,9 @@ private void InitilizeBlobBindings(ExtensionConfigContext context)
             rule.BindToInput<MultiBlobContext>(this); // Intermediate private context to capture state
             rule.AddOpenConverter<MultiBlobContext, IEnumerable<BlobCollectionType>>(typeof(BlobCollectionConverter<>), this);
 
+            // Bind to ParameterBindingData
```

Comment:
nit: unnecessary comment :) Also, regarding your precedence comment, is there actually a precedence issue? It seems that because you're binding to an exact type here, the BindToInput could be anywhere?

Reply:
I can remove the comment but there seems to be a precedence thing happening with input bindings. I originally had `rule.BindToInput<ParameterBindingData>` below `BindToStream` and there was an issue where it was always binding to Stream despite it being ParameterBindingData. When I moved the ParameterBindingData input binding rule above the `BindToStream`, it started to correctly bind to ParameterBindingData when that was the requested type. The stream scenario is still working as expected.

Reply:
There's also already a precedence comment for [BindToStream](https://github.com/Azure/azure-sdk-for-net/blob/46096b131b3d701b821c70912484f77ae05fc612/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs#L109) which led me to believe that this was a thing in the first place

`rule.BindToStream(CreateStreamAsync, FileAccess.ReadWrite); // Precedence, must beat CloudBlobStream`